src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsPage.java

```diff
@@ -0,0 +1,225 @@
+        //  Second date for compare
+        LocalDate secondLocalDate;
+        //  List short name moths
+        List<String> shortNameMonths = new ArrayList<>(Arrays.asList("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"));
+
+        //  Comparison two dates
+        for (int i = 0; i < commentsList.size() - 1; i++) {
+            //  Array of strings with the first comment's date
+            LOGGER.info("First date in String = " + commentsList.get(i).getDateComment().getText());
+            String[] firstDateStrings = commentsList.get(i).getDateComment().getText().split(SPLIT_SYMBOL);
+
+            //  Parse 'firstDateStrings' to LocalDate
+            //  If comment was added recently(during hour)
+            LOGGER.info("'" + firstDateStrings[1] + "' compare to '" + MINUTES + "'");
+            if (firstDateStrings[1].equalsIgnoreCase(MINUTES)) {
+                firstLocalDate = LocalDate.now();
+                LOGGER.info("First date = " + firstLocalDate);
+            } else
+                //  If comment was added recently(during twenty-four hours)
+                LOGGER.info("'" + firstDateStrings[1] + "' compare to '" + HOURS + "'");
+                if (firstDateStrings[1].equalsIgnoreCase(HOURS)) {
+                    //  How many hours ago was added comment
+                    int hourComment = Integer.parseInt(firstDateStrings[0]);
+                    LOGGER.info(hourComment + " " + HOURS + " ago was added comment");
+                    //  What hour at the moment
+                    int hourNow = LocalTime.now().getHour();
+                    LOGGER.info(hourNow + " hour now");
+                    // If comment was added today or yesterday
+                    LOGGER.info(hourComment + " compare to " + hourNow);
+                    if (hourComment <= hourNow) {
+                        LOGGER.info(hourComment + " <= " + hourNow);
+                        firstLocalDate = LocalDate.now();
+                    } else {
+                        LOGGER.info(hourComment + " > " + hourNow);
+                        firstLocalDate = LocalDate.now().minusDays(1);
+                    }
+                    LOGGER.info("First date = " + firstLocalDate);
+                } else {
+                    //  If comment was added more than day ago
+                    //  Day of month
+                    int day = Integer.parseInt(firstDateStrings[0]);
+                    //  Month of year
+                    int month = 0;
+                    //  Year
+                    int year = Integer.parseInt(firstDateStrings[2]);
+
+                    //  Parse name month to number month of year
+                    for (int m = 0; m < shortNameMonths.size(); m++) {
+                        LOGGER.info("'" + firstDateStrings[1] + "' compare to '" + shortNameMonths.get(m) + "'");
+                        if (firstDateStrings[1].equalsIgnoreCase(shortNameMonths.get(m))) {
```

Comment:
Could be replaced with contains and indexOf methods.What if list of month doesn't contains required month?